### DISCUSSION TITLE
Predictions of energies are all same!

### DISCUSSION THREAD

**Discussion by B-C-WANG** [deepmodeling/deepmd-kit#5] [Q&A] [answered]
After training for about 1000 epochs, the energy prediction of all coordinate is a Gaussian distribution, and after training for about 100000 epochs, the energy predictions are all same.
Here is the plot of prediction energy-real energy:
![figure_1](https://user-images.githubusercontent.com/25495799/43062943-f805c118-8e8c-11e8-8e83-5b82fd114346.png)

I think the input coordinate did not encode in a correct way, my system is a 3x3x3 Cu layer with 4 Co and 20 H2O, here is my input json file:

{
    "_comment": " model parameters",
    "use_smooth":	true,
    "sel_a":		[50, 50,50,60,50],
    "rcut_smth":	5.50,
    "rcut":		5.80,
    "filter_neuron":	[25, 50, 100],
    "filter_resnet_dt":	false,
    "n_axis_neuron":	16,
    "n_neuron":		[240, 240, 240],
    "resnet_dt":	true,
    "_comment": " traing controls",
    "systems":		["/opt/deepmd/test/Train"],
    "set_prefix":	"set",    
    "stop_batch":	100000,
    "batch_size":	100,
    "start_lr":		0.005,
    "decay_steps":	5000,
    "decay_rate":	0.95,
    "start_pref_e":	0.02,
    "limit_pref_e":	1,
    "start_pref_f":	1000,
    "limit_pref_f":	1,
    "start_pref_v":	0,
    "limit_pref_v":	0,
    "seed":		1,
    "_comment": " display and restart",
    "_comment": " frequencies counted in batch",
    "disp_file":	"lcurve.out",
    "disp_freq":	100,
    "numb_test":	50,
    "save_freq":	100,
    "save_ckpt":	"model.ckpt",
    "load_ckpt":	"model.ckpt",
    "disp_training":	true,
    "time_training":	true,
    "profiling":	false,
    "profiling_file":	"timeline.json",
    "_comment":		"that's all"
}

**Comment by amcadmus** [comment, marked as answer]
Please try 
"batch_size": 1,

**Comment by B-C-WANG** [comment]
@amcadmus  It works! Thanks!
